extras/vite-plugin-sveltekit-cf-websockets/README.md

```
@@ -0,0 +1,38 @@
+well enough for other use cases as well.
+
+## Usage
+
+Simply add the plugin to your Vite config.
+
+```ts
+import { sveltekit } from '@sveltejs/kit/vite'
+import svelteKitWebSockets from 'vite-plugin-sveltekit-cf-websockets'
+
+export default defineConfig({
+	plugins: [svelteKitWebSockets(), sveltekit()]
```

markdownlint-cli2 (0.20.0)

Hard tabs
Column: 1

(MD010, no-hard-tabs)